package_modules/yum

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import re
+
+
+rpm_cmd = os.environ.get('CFENGINE_TEST_RPM_CMD', "/bin/rpm")
+rpm_output_format = "Name=%{name}\nVersion=%{version}\nArchitecture=%{arch}\n"
+
+yum_cmd = os.environ.get('CFENGINE_TEST_YUM_CMD', "/usr/bin/yum")
+yum_options = "--quiet"
```

Comment:
Shouldn't this be a list, `[ '--quiet' ]` for ease of adding to other lists ?
It's called option**s**, plural, after all ...